Microsoft.Research/Clousot.Cache/SQLCacheModel.cs

```diff
@@ -784,5 +786,33 @@ protected override SqlConnectionStringBuilder BuildConnectionString(IClousotCach
 
       return base.BuildConnectionString(options);
     }
+
+    public static string GetDesiredDbName()
```

Comment:
You could (it's public, and there is a constructor that takes the name), but I didn't want to change the calling code too much - now it's just the original code.